src/style.css

```diff
@@ -0,0 +1,39 @@
+body {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+div {
+  margin: 0 auto;
```

Comment:
I miss the point here... My thought was "to shorten the styling" and "'auto' is OK

Should it be "to avoid changes in unnecessary properties" like the horizontal margins of body?